extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1328,6 +1328,28 @@ describes.realWin('amp-story', {
                 expect(story.activePage_.element.id).to.equal('cover');
               });
         });
+    it('should begin at the specified page fragment parameter value', () => {
```

Comment:
nit: newline before this `it()` and the one below.

Reply:
Done